gems/pending/appliance_console/database_configuration.rb

```diff
@@ -64,6 +64,9 @@ def activate
 
     def post_activation
       ServiceGroup.new.restart_services
+      evm_service = LinuxAdmin::Service.new("evmserverd")
+      evm_service.enable
+      evm_service.start
```

Comment:
cool. so `self` is already in there. if you have the bandwidth, this would be cool. otherwise, just say no and we can merge.